src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class FruitTransaction {
+    private Operation operation;
+    private Fruit fruit;
```

Comment:
(also recommendation, do not change it if no need) 
You use Fruit type to store  only its name, you can just store it in String type or add to your Fruit class also field quantity